server/services/store/sqlstore/migrations/000023_persist_category_collapsed_state.up.sql

```diff
@@ -0,0 +1,29 @@
+{{if .mysql}}
+    SET @dbname = DATABASE();
```

Comment:
This is to allow "CREATE IF NOT EXISTS" in the migration to make this "idenpotent", but... if we put (as we decided) on ALTER/CREATE alone in each migration, you should have that knowledge about idenpotency already. If the transaction succeeded you have the migration applied, so the column is there and the migration is not going to be executed again. If the translaction failed its get not applied at tall. I don't know how much benefit we can obtain here in exchange of readability.

I'm not against this, but looks awful to me.

Reply:
You're right. I completely forgot about this - having only, and one DDL per migration. I'll update the migration.